api/v0/observe/observe.proto

```diff
@@ -52,7 +52,22 @@ service ObserveService {
   rpc GetPosixSignalsStream(GetPosixSignalsStreamRequest) returns (stream GetPosixSignalsStreamResponse) {}
 }
 
-message GetPosixSignalsStreamRequest { 
+/// Request a stream of POSIX signals
+message GetPosixSignalsStreamRequest {
+  /// The workload to which te response will be scoped. If no workload is
+  /// specified, a stream of all POSIX signals on the host will be returned.
+  Workload workload = 1;
```

Comment:
i personally prefer this (with notes) but i think it should be repeated.

Reply:
Should we iterate on the API later? Making this `repeated` warrants some discussion imo. Like how would the response look like in these cases? For a single workload, the response is just a stream of signals. If you would be able to specify multiple cells/containers etc we would need to return tuples (id, signal). If we allow for a mix of workload types we need to return 3-tuples (workload_type, id, signal) for the user to make sense of the response. Maybe something for an RFC?